# Array of legs 
std_msgs/Header header
Leg[] legs
